Lib/test/test_datetime.py

```diff
@@ -0,0 +1,58 @@
+import unittest
+import sys
+
+from test.support.import_helper import import_fresh_module
+
+
+TESTS = 'test.datetimetester'
+
+@unittest.skip("TODO: RUSTPYTHON, incorrect datetime struct implementation in time.rs")
+def load_tests(loader, tests, pattern):
```

Comment:
I don't believe you can skip this. This function is part of the `load_tests` protocol and generates the test suite to be executed. 

If you unskip this what is the error?

Reply:
I can take a look later since am on mobile now. There's probably a reason this file wasn't already added.

Reply:
when it loads _strptime.py I get an error of `no attribute : tzinfo` so I guess ill have to fix that before I can add these. On that topic, could you point me in the right direction for creating a field for the time_struct that may of may not be present for a given instance (naive vs timezone aware time struct)

Reply:
aha, gotcha. Makes sense to remove this file for now and see how the updates play with the rest of the test suite.

> could you point me in the right direction for creating a field for the time_struct that may of may not be present for a given instance

It being an option seems like the way to go, the docs state it as optional and looking through Python code I'm seeing it being guarded by if `tzinfo is None` all over the place.

Reply:
Do you know how I would go about making it so that when the option is none the python object no longer has that attribute? I still don't fully understand how pystructsequence works.